pkg/apis/managed/v1alpha1/subjectpermission_types.go

```diff
@@ -41,8 +41,6 @@ type Permission struct {
 type SubjectPermissionStatus struct {
 	// List of conditions for the CR
 	Conditions []Condition `json:"conditions,omitempty"`
-	// State that this condition represents
-	State string `json:"state,omitempty"`
```

Comment:
This `State` is encapsulated inside the `Conditions` type, and so it is redundant here.

/approve
/lgtm